aggregator/src/tests/rlc/gates.rs

```diff
@@ -76,120 +77,120 @@ impl Circuit<Fr> for ArithTestCircuit {
+                //         let tmp = config.load_private(&mut region, &tmp, &mut offset)?;
+                //         config.decomposition(&mut region, &tmp, 64, &mut offset)?;
+                //     }
+                // }
+                // // unit test: is smaller than
+                // {
+                //     for _ in 0..10 {
+                //         let a = Fr::from(rng.next_u32() as u64);
+                //         let b = Fr::from(rng.next_u32() as u64);
+                //         let c = if a < b { Fr::one() } else { Fr::zero() };
+                //         let a = config.load_private(&mut region, &a, &mut offset)?;
+                //         let b = config.load_private(&mut region, &b, &mut offset)?;
+                //         let c = config.load_private(&mut region, &c, &mut offset)?;
+                //         let c_rec = config.is_smaller_than(&mut region, &a, &b, &mut offset)?;
+                //         region.constrain_equal(c.cell(), c_rec.cell())?;
+                //     }
+
+                //     // equality check
+                //     let a = Fr::from(rng.next_u32() as u64);
+                //     let b = a;
+                //     let c = Fr::zero();
+                //     let a = config.load_private(&mut region, &a, &mut offset)?;
+                //     let b = config.load_private(&mut region, &b, &mut offset)?;
+                //     let c = config.load_private(&mut region, &c, &mut offset)?;
+                //     let c_rec = config.is_smaller_than(&mut region, &a, &b, &mut offset)?;
+                //     region.constrain_equal(c.cell(), c_rec.cell())?;
+                // }
+                // // unit test: is zero
+                // {
+                //     let should_be_false = config.is_zero(&mut region, &f1, &mut offset)?;
+                //     let zero = config.load_private(&mut region, &Fr::zero(), &mut offset)?;
+                //     region.constrain_equal(should_be_false.cell(), zero.cell())?;
+
+                //     let should_be_true = config.is_zero(&mut region, &zero, &mut offset)?;
+                //     let one = config.load_private(&mut region, &Fr::one(), &mut offset)?;
+                //     region.constrain_equal(should_be_true.cell(), one.cell())?;
+                // }
+                // // unit test: is equal
+                // {
+                //     let should_be_false = config.is_equal(&mut region, &f1, &f2, &mut offset)?;
+                //     let zero = config.load_private(&mut region, &Fr::zero(), &mut offset)?;
+                //     region.constrain_equal(should_be_false.cell(), zero.cell())?;
+
+                //     let should_be_true = config.is_equal(&mut region, &f1, &f1, &mut offset)?;
+                //     let one = config.load_private(&mut region, &Fr::one(), &mut offset)?;
+                //     region.constrain_equal(should_be_true.cell(), one.cell())?;
+                // }
```

Comment:
all the commented codes are useless?

Reply:
seems yes

Reply:
those are useful unit tests that were commented out by mistake. they are not uncommented